src/transforms/rotations.jl

```diff
@@ -0,0 +1,82 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Rotation(rotation)
+
+Perform the rotation represented by `rotation`, a
+`ReferenceFrameRotation` object from the `ReferenceFrameRotations` 
+library.
+
+## Examples
+
+```julia
+using ReferenceFrameRotations
+Rotation(ReferenceFrameRotations.EulerAngleAxis(pi/4, [1, 0, 0]))
+```
+
+"""
+struct Rotation{R} <: Meshes.GeometricTransform
+  rotation::R
+end
+
+"""
+    Rotation(axis, α; degrees=true)
+
+Perform the rotation of angle `α` around the axis directed by the 
+vector `axis`. The angle must be given in degrees if `degrees=true`, 
+otherwise in radians.
+
+"""
+function Rotation(axis::Vector{Real}, α::Real; degrees::Bool = true)
+  axisNorm = LinearAlgebra.norm2(axis)
+  if axisNorm == 0
+    error("The `axis` vector is null.")
+  end
+  v = axis / axisNorm
+  if degrees
+    α = α * pi / 180
+  end
```

Comment:
All these checks should be performed already by ReferenceFrameRotations.jl, let them check it for us.

Reply:
I've just checked. There's no check, you can pass a null vector or a non-normalized vector.